storage/encryption/encryption.go

```diff
@@ -79,18 +80,23 @@ func (e *encryption) Decrypt(data []byte) ([]byte, error) {
 	return out, nil
 }
 
-//
+// Reset resets the counter
```

Comment:
Comment reveals an implementation detail. It would be nice to have an explanation in comment when the Reset method must be called, like when reusing the `Encryption`.